### ISSUE TITLE
The icons folder Can't find icon icons/brutus; falling back to default icon

### ISSUE THREAD

**Issue by chenlei00** [beeware/toga#1387]
When using in windows, putting static resource files in the same directory cannot be read, and the error is as follows:Can't find icon icons/brutus; falling back to default icon

**Comment by allaboutmikey** [comment]
The issue seems to be where toga thinks the base folder of the app is. If I put the fully qualified path in, the icons are used.
The problem may be that the app's paths.app call returns the same as paths.toga does. I expected it to be the folder where the application.py file is. On my installation:
```
self.paths.app
WindowsPath('C:/Program Files/Python39/lib/site-packages/toga')
self.paths.toga
WindowsPath('C:/Program Files/Python39/lib/site-packages/toga')
```

**Comment by freakboy3742** [comment]
This is very dependent on how the application is started.

The following three commands:
* `python myapp.py`
* `python -m myapp` (where myapp is a single file)
* `python -m myapp` (where myapp is a folder that contains a `__main__.py`)
* `briefcase dev`

can all be used to start an app - but they'll all give different answers about where the app "is", and what the app is called. If you're getting an answer of `toga` out of `self.paths.app`, I'm betting you're using the first of those invocations. 

I'd recommend you use the third or fourth approach - those provide a lot more context for an app when it's running.

**Comment by freakboy3742** [comment]
Unfortunately, I can't point you at an easy explainer; but it's easy enough to do the experiments yourself. Set up applications structured so that they can be invoked as describe in my last response, and then check:

* Which module is the main module (`sys.modules['__main__']`)?
* Does that module have a package (`sys.modules['__main__'].__package__`)?

You'll get a different set of answers depending on how you invoke your app.

Toga uses these two properties to determine the app name; and when it can't answer the question (which is the case when using `python myapp.py`, it falls back to `toga` as a vaguely helpful default.

As for `app.paths` - you're correct that this is a documentation issue. Our documentation is badly behind where I'd like it to be; fixing this is high on my wish list.

**Comment by freakboy3742** [comment]
@bruno-rino Can you elaborate on your assertion that "images on tutorial2 won't load" if you start with `python -m tutorial`? AFAICT, they work fine in my testing on macOS, Windows and Linux.

**Comment by bruno-rino** [comment]
If you are following the tutorial, you will copy the tutorial code to a file called `something.py`, and start it with `python -m something`. Then the images won't load.

You can see the same effect if you go to the examples folder. If you start the tutorial with `python -m tutorial`, the images load.
But if you `cd tutorial` and then `python -m app`, the images won't load.

**Comment by freakboy3742** [comment]
Ah - got it - it's an issue with using a single file, running as a module.

**Comment by freakboy3742** [comment]
Fixed by #1514.
